portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,32 @@
   width: 650px;
 }
 
-#greeting-container {
+#container {
   margin-top: 20px;
 }
+
+p, li{
+    word-break: keep-all;
```

Comment:
Please keep the indentation consistent: use either 2 spaces or 4 spaces for all appearances.

Reply:
Fixed